CHANGELOG.md

```diff
@@ -5,15 +5,44 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [1.0.0-rc.1] - 2023-12-21
+## [Unreleased]
 
 ### Added
 
 - Warning if running unauthenticated commands as a sudo user
+- Added `sidekick unlink` subcommand to unlink the current project from the
```

Comment:
Nit: if the starting verb is the same as the section title, I tend to leave it out. e.g. look at _Added_ section here: https://keepachangelog.com/en/1.1.0/